src/main/kotlin/com/github/sanity/kweb/KWeb.kt

```diff
@@ -34,19 +43,20 @@ class KWeb(val port: Int,
            val debug: Boolean = false,
            val refreshPageOnHotswap : Boolean = false,
            val plugins: List<KWebPlugin> = Collections.emptyList(),
-           val appServerConfigurator: (AppServer) -> Unit = {},
+           // val appServerConfigurator: (AppServer) -> Unit = {},
```

Comment:
This was more of a temporary way to add additional routes to the Wasabi AppServer, for example it is used by the Select2 plugin - but really need a longer-term  solution for this.